vslib/src/SwitchStateBase.cpp

```diff
@@ -1576,6 +1576,17 @@ sai_status_t SwitchStateBase::refresh_read_only(
             case SAI_SWITCH_ATTR_QOS_MAX_NUMBER_OF_CHILDS_PER_SCHEDULER_GROUP:
                 return SAI_STATUS_SUCCESS;
 
+            case SAI_SWITCH_ATTR_AVAILABLE_IPV4_ROUTE_ENTRY:
+            case SAI_SWITCH_ATTR_AVAILABLE_IPV6_ROUTE_ENTRY:
+            case SAI_SWITCH_ATTR_AVAILABLE_IPV4_NEXTHOP_ENTRY:
+            case SAI_SWITCH_ATTR_AVAILABLE_IPV6_NEXTHOP_ENTRY:
+            case SAI_SWITCH_ATTR_AVAILABLE_IPV4_NEIGHBOR_ENTRY:
+            case SAI_SWITCH_ATTR_AVAILABLE_IPV6_NEIGHBOR_ENTRY:
+            case SAI_SWITCH_ATTR_AVAILABLE_NEXT_HOP_GROUP_MEMBER_ENTRY:
+            case SAI_SWITCH_ATTR_AVAILABLE_NEXT_HOP_GROUP_ENTRY:
+            case SAI_SWITCH_ATTR_AVAILABLE_FDB_ENTRY:
+            case SAI_SWITCH_ATTR_AVAILABLE_ACL_TABLE:
+            case SAI_SWITCH_ATTR_AVAILABLE_ACL_TABLE_GROUP:
```

Comment:
it's not that easy, please take a look at internal company mail, i posted some details how this should be done
then we can discuss whether at this stage this is good enough since i see already SNAT/DNAT entries are already here
but at least default values are set in:
 SwitchStateBase::set_switch_default_attributes() please check this method